src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties

```diff
@@ -21,7 +21,7 @@ javadoc.paragraph.misplaced.tag=<p> tag should be placed immediately before the
 javadoc.paragraph.redundant.paragraph=Redundant <p> tag.
 javadoc.paragraph.tag.after=Empty line should be followed by <p> tag on the next line.
 javadoc.parse.rule.error=Javadoc comment at column {0} has parse error. Details: {1} while parsing {2}
-javadoc.return.expected=Expected @return tag.
+javadoc.return.expected=Expected @return tag with description.
```

Comment:
In issue there is a request  to update message
https://github.com/checkstyle/checkstyle/blob/1cc84e8b8f052249b10a11400a5571cfdaa30e21/src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties#L9

please update it also.

Reply:
I updated issue description with ltest behavior, Check do not report problems with such message.